src/textual/drivers/linux_driver.py

```diff
@@ -120,9 +120,7 @@ def on_terminal_resize(signum, stack) -> None:
         self.console.show_cursor(False)
         self.console.file.write("\033[?1003h\n")
         self.console.file.flush()
-        self._key_thread = Thread(
-            target=self.run_input_thread, args=(asyncio.get_running_loop(),)
-        )
+        self._key_thread = Thread(target=self.run_input_thread, args=(loop,))
```

Comment:
[while I was setting up the integration framework] I noticed that we already have a reference to the event loop in this method, so let's re-use it :-)